audit-api/Chart.yaml

```diff
@@ -27,3 +27,6 @@ dependencies:
   - name: common-lib
     repository: file://../common-lib
     version: 0.1.0
+  - name: fluent-bit
+    repository: https://fluent.github.io/helm-charts
+    version: 0.36.0
```

Comment:
Add a tag alone with a default value = true in values file so fluent-bit is installed by default but could be disabled if needed.

e.g.
```
tags:
   - fluent-bit
```

and in values.yaml
```
tags:
  fluent-bit: true
```